File: app/src/main/java/com/example/watering/investrecord/CardDBHelper.java
package com.example.watering.investrecord;

import android.content.Context;

/**
 * Created by watering on 17. 10. 21.
 */

@SuppressWarnings({"ALL"})
class CardDBHelper extends DBHelper {

    public CardDBHelper(Context context) {
        super(context);

        TABLE_NAME = "tbl_card";
        COLUMNS = new String [] {"number TEXT UNIQUE","company TEXT","name TEXT","date_draw TEXT"};
    }
}

File: app/src/main/java/com/example/watering/investrecord/DBHelper/CardDBHelper.kt
package com.example.watering.investrecord.DBHelper

import android.content.Context

/**
 * Created by watering on 17. 10. 21.
 */

class CardDBHelper(context: Context) : DBHelper(context) {

    init {
        TABLE_NAME = "tbl_card"
        COLUMNS = arrayOf("number TEXT UNIQUE", "company TEXT", "name TEXT", "date_draw TEXT", "id_account INTEGER")
    }
}
